theforeman.org/scripts/debian/setup_sources_core.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+set -xe
+
+# This script clones the git repos, cleans them up, and builds the package
+echo "--Setting Up Sources"
+
+# Setup the debian files, figure out the version
+echo `git log -n1 --oneline`
+cd debian/${os}/
+VERSION=$(head -n1 ${project}/changelog|awk '{print $2}'|sed 's/(//;s/)//'|cut -f1 -d-)
+
+# Setup sources
+mkdir build-${project} && cd build-${project}
+if [[ x$repo =~ ^xdevelop ]]; then
+  url_base='https://ci.theforeman.org'
+  job_name=$nightly_jenkins_job  # e.g. 'test_develop', from triggering job
+  job_id=$nightly_jenkins_job_id # e.g. '123', from triggering
+  json_url="${url_base}/job/${job_name}/${job_id}/api/json"
+
+  # If a last* alias was used, resolve the numeric job ID
+  job_id=$(curl "${json_url}" | /usr/local/bin/JSON.sh -b | awk '$1 == "[\"number\"]" { print $2 }')
+  base_url=`curl "${json_url}" | /usr/local/bin/JSON.sh -b | awk '$1 ~ /^\["runs",.*,"number"\]/ && $2 == '$job_id' {getline; print $2; exit}' | tr -d \"`
+  if [ x$base_url = x ] ; then
+    base_url=`curl "${json_url}" | /usr/local/bin/JSON.sh -b | egrep '\["url"\]' | awk '{print $NF}' | tr -d \"`
+  fi
+  url="${base_url}/artifact/*zip*/archive.zip"
+
+  wget $url
+  unzip archive.zip
+  mv archive/pkg/*bz2 ${project}_${VERSION}.orig.tar.bz2
+
+  # Set this in case we need it
+  LAST_COMMIT=`curl "${json_url}" | /usr/local/bin/JSON.sh -b | egrep '"lastBuiltRevision","SHA1"' | awk '{print $NF}' | tr -d \" | head -n1`
+else
+  VERSION=`echo ${VERSION} | tr '~rc' '-rc'`
+  # Download sources
+  wget http://downloads.theforeman.org/${project}/${project}-${VERSION}.tar.bz2
+  wget http://downloads.theforeman.org/${project}/${project}-${VERSION}.tar.bz2.sig
```

Comment:
While we're at it, can we change this to
```suggestion
  wget https://downloads.theforeman.org/${project}/${project}-${VERSION}.tar.bz2 https://downloads.theforeman.org/${project}/${project}-${VERSION}.tar.bz2.sig
```